indexer/etl/l1_etl.go

```
@@ -131,7 +131,14 @@ func (l1Etl *L1ETL) Start(ctx context.Context) error {
 						return err
 					}
 					return nil
-				}); err != nil {
+				})
+
+				if err != nil && strings.Contains(err.Error(), "duplicate key value violates unique constraint") {
```

I don't think we should silently ignore this as it's indicative of other errors, rather always ensure that the indexer is never re-indexing the same data.

---

d605488